.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: php
+
+php:
+  - 7.0
+  - 7.1
+
+sudo: false
+
+cache:
+  directories:
+    - cache
+    - vendor
+    - $HOME/.composer/cache
+
+install:
+  - npm install
+  - composer install --no-interaction --prefer-source
+
+before_script:
+  - if find . -name "*.php" ! -path "./vendor/*" -exec php -l {} \; | grep "unexpected $end"; then exit 1; fi
+
+script:
+  - echo 'scripts'
```

Comment:
This is where we will add phpunit test commands. [Reference](https://github.com/travis-ci-examples/php/blob/master/.travis.yml)